resources/tests.mjs

```diff
@@ -248,8 +248,8 @@ Suites.push({
 });
 
 Suites.push({
-    name: "VueJS-TodoMVC",
-    url: "todomvc/architecture-examples/vuejs-cli/dist/index.html",
+    name: "TodoMVC-Vue-Webpack",
```

Comment:
Why rename to include "webpack" (in the suite name but also the file structure)? My understanding from https://docs.google.com/document/d/1MXxk9DryJmvAmPMt2uMneaS3J5Y-dMFGt4EGvMzhhJE/edit#heading=h.175iv9aumkej is that we aren't, for example, planning to include multiple variations based on build tooling.

Reply:
just personal naming preference - removing webpack from test name 👍

Reply:
done